app/api/currentuser/get_info.go

```diff
@@ -52,7 +52,7 @@ func (srv *Service) getInfo(w http.ResponseWriter, r *http.Request) service.APIE
 		AllowSubgroups    *bool   `gorm:"column:allowSubgroups" json:"allow_subgroups"`
 	}
 
-	err := srv.Store.Users().ByID(user.UserID).
+	err := srv.Store.Users().ByID(user.ID).
```

Comment:
The test coverage for this file I think. And probably it is because your "user not existing" scenarios are now covered by the middleware. So probably `err == gorm.ErrRecordNotFound` is irrelevant now. But you should add a short comment explaining the `gorm.ErrRecordNotFound` is irrelevant in the case as are sure that the user exists because of the middleware.
... hum actually that's not true as, because of race condition, the user may have been deleted in between. But you should still have to add a comment saying it is very unlikely... and still find a way to test it :-)

Reply:
```	if err == gorm.ErrRecordNotFound {
		return service.InsufficientAccessRightsError
	}
	service.MustNotBeError(err)```